packages/fluentui/react-northstar/src/components/Chat/ChatItem.tsx

```diff
@@ -135,7 +134,7 @@ export const ChatItem = (React.forwardRef<HTMLLIElement, ChatItemProps>((inputPr
     });
 
     return (
-      <ChatItemContextProvider value={{ attached }}>
+      <ChatItemContextProvider value={{ attached, unstable_layout: layout }}>
```

Comment:
The is no need to memoize this context, but you will need to use `useContextSelector()` from `@fluentui/react-bindings` to get a value from it.

Reply:
Thanks.